cfg/validate.go

```diff
@@ -245,6 +246,23 @@ func isValidBufferedReadConfig(rc *ReadConfig) error {
 	return nil
 }
 
+func isValidLoggingConfig(loggingConfig *LoggingConfig) error {
+	if loggingConfig == nil {
+		return nil
+	}
+
+	const DefaultLoggingFormat = LoggingFormatJson
```

Comment:
nit: Could you move this also to the `const()` section at the top ?

```suggestion
	const DefaultLoggingFormat = LoggingFormatJson
```